tests/queries/0_stateless/02498_storage_join_key_positions.sql.j2

```diff
@@ -1,80 +0,0 @@
--- Tags: no-parallel-replicas
```

Comment:
Why are these two tests are removed?

Reply:
I rechecked and realized that I introduced a non-`.j2` version of the test in the previous PR but forgot to remove it there. (Since using jinja here is pointless)

https://github.com/ClickHouse/ClickHouse/blob/master/tests/queries/0_stateless/02498_storage_join_key_positions.sql